packages/nextjs/components/scaffold-eth/Contract/NinjaUI.tsx

```diff
@@ -0,0 +1,93 @@
+import { Contract } from "ethers";
```

Comment:
Yeah, let's remove this. As in the [contributing guidelines](https://github.com/scaffold-eth/se-2/blob/master/CONTRIBUTING.md), we want to keep the core simple an minimal.